tests/cypress/integration/issues_prs2/issue_3821_delete_point.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('When delete a point, the required point is deleted.', () => {
+    const issueId = '3821';
+
+    const firstPointsShape = {
+        labelName,
+        type: 'Shape',
+        pointsMap: [
+            { x: 300, y: 250 },
+            { x: 300, y: 350 },
+            { x: 300, y: 450 },
+        ],
+    };
+    const secondPointsShape = {
+        labelName,
+        type: 'Shape',
+        pointsMap: [
+            { x: 330, y: 250 },
+            { x: 330, y: 350 },
+            { x: 330, y: 450 },
+        ],
+    };
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    describe(`Testing issue "${issueId}"`, () => {
+        it('Crearte points shapes', () => {
+            cy.createPoint(firstPointsShape);
+            cy.get('#cvat-objects-sidebar-state-item-1').should('contain', '1').and('contain', 'POINTS SHAPE');
+            cy.createPoint(secondPointsShape);
+            cy.get('#cvat-objects-sidebar-state-item-2').should('contain', '2').and('contain', 'POINTS SHAPE');
```

Comment:
Probably it should be a part of ``before`` section because we do not test creating points here